clients/ui/frontend/src/app/utils/string.ts

```diff
@@ -0,0 +1,9 @@
+export const genRandomChars = (length = 8): string => {
```

Comment:
we have this util in shared library, we can import it from there. so we can remove this file